public/js/console-game.js

```diff
@@ -264,6 +278,6 @@ var Game;
     basicActions.go   = basicActions.move   ;
     basicActions.take = basicActions.pickup ;
     
-    this.moveTo("entry");
+    return this.moveTo("entry");
   }
 })();
```

Comment:
I'd say something about the lack of new line at the end of the file... and since i am... well, anyway, after this merge i'll Browserify all this jazz so we can have lots of great separate files for all the things.